flask_admin/contrib/sqla/filters.py

```diff
@@ -118,6 +120,38 @@ def operation(self):
         return lazy_gettext('not in list')
 
 
+# Composite filters
+class FilterAndEqual(FilterEqual):
```

Comment:
It's the same as adding "n" filters, but basically yes.
I think it's sometimes easier for the user to use one filter if they have some kind of identifier they could paste, instead of opening up multiple filters.